packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -281,6 +288,14 @@ export class DocumentService
 		// Retry with new token on authorization error; otherwise, allow container layer to handle.
 		try {
 			const connection = await connect();
+			if (
```

Comment:
I'd suggest to always update policy, on every connection. I.e. allow true -> false transitions - service might want to pull back the flight.

Reply:
Fixed